docs/migration/manual-migration.mdx

```diff
@@ -58,6 +58,17 @@ mechanics from Remix loaders and hooks:
   transfered to `useTrackPage` and `useProductPage` hooks
 - Loading state management is done directly at route loader level
 
+#### `Account` pages
+
+Similarly, page-base HOCs on the Account page have been completely removed, and
+replaced with standard Remix and React mechanics. Here's a list of all removed
+Enhancers in those pages:
+
+- `EnhanceDownloadableProduct` (see
+  [merge request](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/merge_requests/2444))
+- `EnhanceAddressBook` (see
+  [merge request](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/merge_requests/2439))
```

Comment:
As mentioned in those MR's I don't think we should remove these enhancers, only the graphql queries in the enhancers which are now handled by the loaders, otherwise it introduces a breaking change for no reason? unless there was an issue using the enhancers.

Reply:
Since the enhancer has been deleted too after discussions, is it ok for you to close this issue ?